File: src/main/java/net/slipp/support/wiki/WikiContents.java
package net.slipp.support.wiki;

import org.pegdown.Extensions;
import org.pegdown.PegDownProcessor;

public class WikiContents {
	public static String parse(String contents) {
		return new PegDownProcessor(Extensions.FENCED_CODE_BLOCKS).markdownToHtml(contents);
	}
}

File: src/main/scala/net/slipp/support/wiki/WikiContents.scala
package net.slipp.support.wiki

import org.pegdown.Extensions
import org.pegdown.PegDownProcessor

object WikiContents {
  def parse(contents: String): String = {
    return new PegDownProcessor(Extensions.FENCED_CODE_BLOCKS).markdownToHtml(contents)
  }
}

